cni/netconfig.go

```diff
@@ -40,43 +40,45 @@ type RuntimeDNSConfig struct {
 	Options  []string `json:"options,omitempty"`
 }
 
+type Ipam struct {
```

Comment:
That looks much better pulled out of `NetworkConfig`

Reply:
ah missed that when I was moving things around, I'm usually a stickler for capitalizing acronyms 😆